### ISSUE TITLE
Move components to public site

### ISSUE THREAD

**Issue by emilyjablonski** [bloom-housing/bloom#3238]
We should move all of these components out of UIC and into the public site. All of these components are currently only used on the public site, and a number of them include either/both a lot of layout or Bloom-specific business logic.

The below components are used only once.
`ApplicationTimeline`
`GetApplication`
`StatusItem`
`SubmitApplication`
`DownloadLotteryResults`
`HouseholdMemberForm`

The below component is used more than once.
`HouseholdSizeField`

We should move all of them to a components/ folder in the public site that will house components that we use just once, and ones that we reuse.

I would suggest the following folder structure all nested under a src folder (so that all site content is in once place - next should already support the pages directory living within src), where the components folder is organized by page structure for components used only once.

pages/
--account/
--applications/
--listing/
----[id].tsx

components/
--account/
--applications/
--listing/
----GetApplication.tsx
----GetApplication.scss
----GetApplication.test.tsx
--shared/ (components used in more than one file)
----HousholdSizeField.tsx
----HousholdSizeField.scss
----HousholdSizeField.test.tsx

AC:
Public site has the new directory structure. Will require moving everything under src, organizing the existing components that live in the src directory into the new components directory, copying and pasting the listed components out of UIC into these folders, and importing these versions instead of the UIC versions where they are used.

Near future work separated into different ticket:
Run unit tests in the components folder

Try to move unit tests over, but if it causes issues, create new ticket to spin it out

**Comment by emilyjablonski** [comment]
QA Notes: The only code change was in the StatusItem component, since it was moved to use CSS modules (this component renders on the my applications page within accounts and is a summary of a submitted application). I would ensure there are not visual regressions by submitting an application and checking out the component.
